### ISSUE TITLE
make: golangci-lint: No such file or directory

### ISSUE THREAD

**Issue by kenorb** [FunctionX/fx-core#7]
```
$ make
...
--> Running linter
golangci-lint run
make: golangci-lint: No such file or directory
make: *** [Makefile:174: lint] Error 127
```

How to install linter?

**Comment by zakir-code** [comment]
https://golangci-lint.run/usage/install/

**Comment by kenorb** [comment]
Should this be added to README?

**Comment by kenorb** [comment]
### Ubuntu

```
sudo snap install golangci-lint
```

**Comment by kenorb** [comment]
- Should this be added to README?
- Also, shouldn't linter be optional, not part of `make`? As currently `make` command fails due to lot of typecheck errors.

**Comment by zakir-code** [comment]
* It can be considered, it is useful for developers. 
* Thank you very much for your feedback, we will fix it as soon as possible

**Comment by zakir-code** [comment]
Run `make` by default to execute `make all`, you can find it from the `makefile`, and then it will execute `make install`, `make lint` and `make test` in order. At present, there is an error in the process of executing `make lint`, but your `fxcored` program is installed succeeded

**Comment by zakir-code** [comment]
this has been fixed on 9fe9dea8474d5540cb649b558add3531faf37b5f

**Comment by kenorb** [comment]
Works fine:

```
$ go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.43.0
$ make lint
--> Running linter
golangci-lint run
find . -name '*.go' -type f -not -path "./vendor*" -not -path "*.git*" -not -name '*.pb.*' | xargs gofmt -d -s
```
